src/main/java/com/iexec/common/replicate/ReplicateStatusCause.java

```diff
@@ -79,6 +79,7 @@ public enum ReplicateStatusCause {
     TEE_SESSION_GENERATION_NO_SESSION_REQUEST,
     TEE_SESSION_GENERATION_NO_TASK_DESCRIPTION,
     TEE_SESSION_GENERATION_GET_SESSION_FAILED,
+    TEE_SESSION_GENERATION_NO_SMS_FOR_TASK,
```

Comment:
Is it used somewhere ? It does not seem to be the case.

Reply:
You're right, it is not used anymore. Please see https://github.com/iExecBlockchainComputing/iexec-sms/pull/116/commits/7fc445e18f2aaad1c99e3919820f5fc580b73c30.